### DISCUSSION TITLE
Which model is the best for quickest faceswap with complex text prompt ?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2951] [Q&A]
I tried prompt as "female model, luxury fashion photoshoot, stunning Paris, Summer, Haussmann-style buildings, intricate architecture, tree-lined boulevards, leaves, the Eiffel Tower in the distance, and the blue Seine flows under bridges"

But the it takes around 40s to generate one image and quality is bad, missing background styles.
I need help which model, which config is needed for it.
![image](https://github.com/lllyasviel/Fooocus/assets/91205658/64cdf458-fd85-461c-8194-fa874edbd1a0)

**Comment by mashb1t** [comment]
To get facts straight: the model normally has nothing to do with the speed (except Turbo).
Every SDXL generalistic or photographic model should do the trick.
Please check out the following steps to improve image quality:

- generate image
- upscale
- inpaint

OR

- generate image
- (upscale)
- face swap
